pkgs/development/libraries/dbus/default.nix

```diff
@@ -13,6 +13,11 @@
 , docbook_xml_dtd_44
 , docbook-xsl-nons
 , xmlto
+
+, withStatic ? false
```

Comment:
Maybe default to `stdenv.hostPlatform.isStatic` instead?

Reply:
Didn't know this flag existed, thanks !